### PR TITLE
Model refactoring based on feedback (Session 1)

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
# Committer Notes

Work on multiple issues described by the epic #597.

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

**Comment by david-waltermire-nist** [comment]
I mistakenly referenced #518 when I intended to reference #581.

File: src/metaschema/oscal_metadata_metaschema.xml [hunk 1]
```diff
@@ -93,19 +99,42 @@
          </remarks>
          <o:profile>
             <o:metadata>
-              <o:link rel="reference" href="#citation1">My citation</o:link>
+              <o:link rel="citation" href="#resource1">My citation</o:link>
```
Comment (bradh) [inline src/metaschema/oscal_metadata_metaschema.xml]: This doesn't appear at https://www.iana.org/assignments/link-relations/link-relations.xhtml (although the previous one didn't either).

Comment (david-waltermire-nist) [reply src/metaschema/oscal_metadata_metaschema.xml]: We use rels that are not in the link relations IANA registry. I think we need to come up with a means for the IANA relations and the OSCAL relations to co-exist.

**Comment by david-waltermire-nist** [comment]
PR #619 includes these changes. Closing this PR.

**Pull request closed without merging**